osde2e-wrapper.py

```diff
@@ -12,21 +12,21 @@
 #   limitations under the License.
 
 import argparse
-from datetime import datetime
-from elasticsearch_dsl import Search 
+# from datetime import datetime
```

Comment:
If we don't need these lets just remove them instead of leaving them commented out.